### PR TITLE
Add API cli feature

### LINKED ISSUE #53: Add api CLI feature

### PR DESCRIPTION
**Pull request by vkehfdl1**
close #53

We can now run api server with CLI command

```bash
autorag run_api --config_path your/path/to/yaml_file.yaml --host 127.0.0.1 --port 8000
```

merge this after merge #52

### CODE REVIEW

File: autorag/evaluator.py [hunk 1]
```diff
@@ -155,7 +156,18 @@ def evaluate(config, qa_data_path, corpus_data_path):
     logger.info('Evaluation complete.')
 
 
-cli.add_command(evaluate)
+@click.command()
+@click.option('--yaml_path', type=str, help='Path to extracted yaml file.')
```
Comment (Eastsidegunn) [inline autorag/evaluator.py:160]: Of course, the only Yaml file format used in our library is the pipeline, but I would like you to mention that it is a variable about the pipeline in the description of the variable.

I think it's hard to read the code internally to make the variable name as Pipeline

Comment (vkehfdl1) [reply autorag/evaluator.py:160]: How about config_path?

**Comment by Eastsidegunn** [comment]
changed example according to conversation
`autorag run_api --confilg_path your/path/to/yaml_file.yaml --host 127.0.0.1 --port 8000`

**Review by Eastsidegunn — DISMISSED** [review]

**Review by bwook00 — DISMISSED** [review]
LGTM

**Review by bwook00 — APPROVED** [review]
LGTM

**Review by Eastsidegunn — APPROVED** [review]

**Pull request merged**